react-debugging.html

```diff
@@ -39,28 +42,34 @@
         this.setState({ setMood: this.state.moods[this.state.currentMood] })
       }
 
-      render(){
+      render() {
         return(
+          //Fix #3: put the return into one div container
+          <>
             <h1>Welcome to the Mood Meter App</h1>
             <h3>Mood Options:</h3>
             <p style={{fontSize: "50px"}}>
               { this.state.moods[this.state.currentMood] }
             </p>
-            <button onClick={ }>
+            <button onClick={ this.nextMood() }>
               Click Here For Next Mood Option
             </button>
             <h3>Save your mood here!</h3>
-            <button onClick={ this.setThisMood }>
+            <button onClick={ this.setThisMood() }>
```

Comment:
Since its calling on the method above, the argument braces are not required in React.